loader/src/sources/albertsons/index.js

```diff
@@ -206,18 +213,69 @@ async function getData(states) {
       return formatted;
     })
     .filter((location) => states.includes(location.state));
+
+  // Adult & Pediatric vaccines at the same locations get different listings in
+  // the Albertsons API, so we need to group and merge them.
+  const groups = _.groupBy(formatted, (location) => {
+    return (
+      location.external_ids.find(
+        (id) => id[0] === "albertsons_store_number"
+      )?.[1] ?? location.name
```

Comment:
`??`, razzle dazzle!

I will admit, it took me many re-reads to grok all this is doing. Since it's already multiline, maybe worthwhile being less terse?

```
const storeExtId = location.external_ids.find((id) => id[0] === "albertsons_store_number");
return storeExtId ? storeExtId[1] : location.name;
```

or, since you have lodash now

```
const storeExtId = location.external_ids.find(_.matches({0: "albertsons_store_number"}));
return storeExtId ? storeExtId[1] : location.name;
```

Reply:
Good point. That said, I think this mainly looks confusing because of Prettier trying to shorten the lines, which still happens even if we separate them.  :\

```js
const storeId = location.external_ids.find(
      (id) => id[0] === "albertsons_store_number"
    );
    return storeId ? storeId[1] : location.name;
```

But it’s definitely clearer than what I’ve got now.